app/js/lib/foxbox/services/base.js

```diff
@@ -112,34 +114,72 @@ export default class BaseService extends EventDispatcher {
   /**
    * Setups value watcher for the getter matching specified selector.
    *
-   * @param {Object} selector Selector to match getter which value we would like
-   * to watch.
+   * @param {string} alias Watcher alias to match getter which value we would
+   * like to watch.
    * @param {function} handler Function to be called once getter value changes.
    */
-  watch(selector, handler) {
+  watch(alias, handler) {
```

Comment:
Nice, thanks! At this stage we have that luxury to experiment without breaking something deployed in production :)